src/main/java/com/spotinst/sdkjava/model/api/ocean/ecs/ApiClusterOptimizeImageConfiguration.java

```diff
@@ -0,0 +1,71 @@
+package com.spotinst.sdkjava.model.api.ocean.ecs;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class ApiClusterOptimizeImageConfiguration {
+
+    private Set<String>  isSet;
+    private Boolean      shouldOptimizeEcsAmi;
+    private String       performAt;
```

Comment:
Recheck if this parameter should be String or enumeration.

Reply:
This should be enum. THis file is shared by Bansi and it is not required for launch specification. Leaving as it is. @chaplabansi will make the necessary changes.